drake/util/TrigPoly.h

```diff
@@ -26,6 +26,10 @@
  * cout << sin(x) * x                 // emits "x1*s1"
  * cout << sin(x + x) * x             // emits "x1*s1*c1 + x1*c1*s1"
  * \endcode
+ *
+ * NOTE:  Certain analyses may not succeed when individual Monomials contain
+ * both x and sin(x) or cos(x) terms.  This restriction is not currently
+ * enforced programmatically (TODO ggould).
```

Comment:
I think we're using the TODO(ggould) form of TODO comments as described here https://google.github.io/styleguide/cppguide.html#TODO_Comments